### PR TITLE
Added opcahce.php file for webserver (SNMP)

### PR DESCRIPTION
**Pull request by SniperVegeta**

### CODE REVIEW

File: snmp/opcache [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
```
Comment (Jellyfrog) [inline snmp/opcache]: This should be
```suggestion
#!/usr/bin/env php
<?php
```

File: snmp/opcache [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# This is the status script that goes with the Cacti Templates for opcache via SNMP
+# It can be accessed as a snmpd extension with something like:
+#       extend phpopcache /usr/bin/curl --silent http://localhost/opcache.php
```
Comment (Jellyfrog) [inline snmp/opcache]: This comment is wrong, it should be a path to the script directly.
Like:
```suggestion
#       copy this script to /etc/snmp/
#       chmod +x /etc/snmp/osupdate  
#       edit your snmpd.conf and include: extend phpopcache /etc/snmp/opcache
```

File: snmp/opcache [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# NOTE you should put an htaccess file on it for protection
+# need to be surved true the webserver for correct stats / CLI = other stats
+
+header('Content-Type: text/plain');
```
Comment (Jellyfrog) [inline snmp/opcache]: can be removed

**Comment by SniperVegeta** [comment]
Well, i try that but, the CLI got other stats then true the webserver so thats why i curl to the php script inside the webserver....

Via the cli its not working....

**Comment by SourceDoctor** [comment]
what is this for? which Application uses this?

**Pull request closed without merging**